components/event-stream/org.wso2.carbon.event.stream.ui/src/main/resources/web/eventstream/js/event_stream.js

```diff
@@ -248,6 +264,10 @@ function addStreamAttribute(dataType) {
         error = "Attribute type field is empty. \n";
     }
 
+    if ((attributeName.value).indexOf(" ")!=-1 && (attributeName.value).indexOf(" ")!= (attributeName.value).length-1){
```

Comment:
There seems to be some issues in code formatting. Can you fix all formatting issues please.